modules/pagereader.py

```diff
@@ -44,10 +44,15 @@ def read_first_page(site, extension=False):
             attempts_left -= 1
             err = e
 
-    if err == HTTPError:
-        raise("There has been an HTTP error after three attempts.")
-    if err == ConnectionError:
-        raise("There has been a connection error after three attempts.")
+    if type(err) == HTTPError:
+        print ("There has been an HTTP error after three attempts.")
+        #exit 2 to indicate that there has been exception
+        exit (2)
```

Comment:
I meant could you explain what the error code indicates. For example, an error code of 2 in Unix systems usually symbolizes a command line syntax error. We should probably use an error code of 1 because that is used for most other applications. 

Check: https://docs.python.org/3/library/sys.html#sys.exit